internal/wasi_snapshot_preview1/logging/logging.go

```diff
@@ -14,11 +14,13 @@ import (
 
 var le = binary.LittleEndian
 
-func IsFilesystemFunction(fnd api.FunctionDefinition) bool {
+func IsInLogScope(fnd api.FunctionDefinition, scopes logging.LogScopes) bool {
```

Comment:
I might keep ` IsFilesystemFunction` as `isFilesystemFunction` then dispatch to it from `IsInLogScope`. This should keep the logic easier.

so IsInLogScope would be a case statement for the known scopes, which does a `fallthrough` each time as they can compose (watch the edge case of filesystem|crypto)